NEWS.md

```diff
@@ -1,3 +1,7 @@
+## 3.1.1 2021-12-15
```

Comment:
Please remove the date (and replace with IN-PROGRESS) as it will be incorrect until the module is released